src/components/App/App.tsx

```diff
@@ -21,11 +26,47 @@ class App extends React.Component<Props> {
     .then(data => this.setState({apiList: data.entries})) 
   }
 
+  filter = (stateObj: FilterState):any => {
+    let matchingCards: Api[] = [];
+    if (this.state) {
+            matchingCards = this.state.apiList.filter(api => {
+      return api.API.includes(stateObj.search)
+    })
+      stateObj.Categories && (
+        matchingCards = matchingCards.filter(api => {
+          return api.Category.toLowerCase() === stateObj.Categories.toLowerCase()
```

Comment:
does this need .toLowerCase since we are pulling from the category

Reply:
Yes for now because the value of the category inputs don't all match up with the category props on the data